README.md

```diff
@@ -122,13 +122,44 @@ kind: Service
 metadata:
   annotations:
     kube2lb/port-mode: |
-      { "mysql": "tcp" }
+      { "mysql": "tcp", "healthz": "http" }
```

Comment:
Not related. Modified example to have multiple ports, as it has caused confusion in the past.